lib/rss.js

```diff
@@ -0,0 +1,192 @@
+const RSS = require('rss')
+const generateDiff = require('deep-diff').diff
+const fs = require('fs')
+
+function createLeadersHash(array) {
+  return array.reduce((hash, elem) => {
+    hash[elem.id] = elem
+    return hash
+  }, {})
+}
+
+function createOrgHash(array) {
+  let rank = 0
+  return array.reduce((hash, elem) => {
+    rank++
+    elem.rank = rank
+    elem.leaders = createLeadersHash(elem.leaders)
+    hash[elem.slug] = elem
+    return hash
+  }, {})
+}
+
+module.exports = async ({
+  data: newData,
+  data_old: oldData,
+  feed_items: current,
+}) => {
+  const dataUpdated = fs.statSync(`${__dirname}/../out/data.json`).mtime
```

Comment:
This can come from the arg

Reply:
I didn't want to bloat the main script too much but sure I can do the stat from the main script.